tests/base_test.py

```diff
@@ -497,6 +497,17 @@ def test_rule_changes(ea):
     mock_load.assert_any_call('rules/rule2.yaml')
     mock_load.assert_any_call('rules/rule3.yaml')
 
+    # A new rule with a conflicting name wont load
+    new_hashes = copy.copy(new_hashes)
```

Comment:
This dict is of the form {filename: hash, filename2: hash2, etc}. I think normal copy is ok in this case.